src/main/java/seedu/taskell/model/task/TaskTime.java

```diff
@@ -0,0 +1,56 @@
+package seedu.taskell.model.task;
+
+
+import seedu.taskell.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Task's taskTime in the task manager.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTaskTime(String)}
+ */
+public class TaskTime {
+
+    public static final String MESSAGE_TASK_TIME_CONSTRAINTS =
+            "Task time should be 2 alphanumeric/period strings separated by '@'";
+    public static final String TASKTIME_VALIDATION_REGEX = "[\\w\\.]+@[\\w\\.]+";
+
```

Comment:
The other taskDate,taskPriority classes are TASK_DATE... Preferable TASK_TIME for consistency